mizar/get_interface.py

```diff
@@ -0,0 +1,11 @@
+import os
+import logging
+logger = logging.getLogger()
+
+def get_itf():
+    default_itf = os.popen("route | grep '^default' | grep -o '[^ ]*$'").read().split('\n')[0]
```

Comment:
Not sure if "route" command is always available. We probably can set default_itf to "eth0" if the execution and parsing result is empty?

Reply:
We can set it to 'eth0' but, it will work again as current the scenario of the eth0 bug. As the interface name is generally changed with the version of Ubuntu OS. In ubuntu 16.0 it was 'eth0' but in ubuntu 18.0 and above the interface name changed to something like 'enXXX' due to the Predictable Interface Names.

Reply:
We can set it to 'eth0' but, it will work again as current the scenario of the eth0 bug. As the interface name is generally changed with the version of Ubuntu OS. In ubuntu 16.0 it was 'eth0' but in ubuntu 18.0 and above the interface name changed to something like 'enXXX' due to the Predictable Interface Names.